content/rs/administering/access-control/password-rotation.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: Database Password Rotation
```

Comment:
It sounds like this features applies to password rotation in general, not just database use password rotation. Is that right? If so, is this the correct title?

Reply:
That's true but the main use case is the database password. Alon didn't want to highlight the user password rotation.